samples/bot-adaptive-card-actions/csharp/README.md

```diff
@@ -0,0 +1,128 @@
+---
+page_type: sample
+description: Demonstrating the feature where user can send adaptivecard actions using bot.
```

Comment:
Demonstrating the feature where user can send adaptive card with different actions using bot.

Reply:
fixed